.gitignore

```diff
@@ -1,6 +1,9 @@
 *.rbc
+mix.lock
```

Comment:
... won't that make it fail whenever a new version comes out and then whoever wanted to modify something unrelated has to figure out a elixir issue ?